CONTRIBUTING.md

```diff
@@ -50,3 +50,18 @@ Before the publish it runs the `npm run build` script which generates type defin
 The script also generates the API docs. After the publish, the API docs are automatically pushed to the `gh-pages` branch which updates the documentation at  [octokit.github.io/rest.js](https://octokit.github.io/rest.js).
 
 If the pull request looks good but does not follow the commit conventions, use the "Squash & merge" button.
+
+## Testing a pull request from github repo locally:
```

Comment:
I think I would move the new segment above "Merging the Pull Request & releasing a new version", what do you think? Chronologically it makes more sense that way to me

Reply:
True, that's a good catch! Changed it 😄